### PR TITLE
* Circular Progress Bar

### PR DESCRIPTION
**Pull request by Wagnerp**
- Need demo

### CODE REVIEW

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿using Krypton.Toolkit;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs:1]: Sorting ?

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+using System;
+using System.Collections.Generic;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs:3]: I'm sure all of these are not needed for this file.

File: Source/Krypton Toolkit/Examples/CircularProgressBar/CircularProgressBar.csproj [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net5.0-windows</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="krypton.toolkit.canary.lite" Version="6.0.2103.6" />
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/CircularProgressBar/CircularProgressBar.csproj:11]: If this is for Extended, then do not include the toolkit standard here.

**Comment by Smurf-IV** [comment]
The third party anumation library needs to havea license file / heraders added.
The Circular prject needs to have 
- the Licenses header added
- A readme.md file started (To add in the Demo gif and explanation of what the paramters do)

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs:1]: My installation was sorting the statements, only found out after checking this in. Will solve this.

File: Source/Krypton Toolkit/Examples/CircularProgressBar/CircularProgressBar.csproj (hunk 3 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/CircularProgressBar/CircularProgressBar.csproj]: The issue is that I don't appear to have access to the extended toolkit controls from the toolbox, even if I build the project.

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs (hunk 1 above)
Comment (Smurf-IV) [reply Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs:1]: Need the System.### ones first. (It's a setting in the Visual Studio Options)

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Readme.md [hunk 4]
```diff
@@ -0,0 +1 @@
+﻿
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/CircularProgressBar/Readme.md:1]: Add reference to this, via the start document

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs [hunk 5]
```diff
@@ -10,5 +18,45 @@ public Form1()
...
+            tmrIncrement.Enabled = false;
+
+            ccbTest.Text = "0%";
+        }
+
+        private void tmrIncrement_Tick(object sender, EventArgs e)
+        {
+            ccbTest.Increment(1);
+
+            ccbTest.Text = $"{ ccbTest.Value }%";
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs:37]: Do not put spaces around the variable

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Readme.md (hunk 4 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/CircularProgressBar/Readme.md]: Done

File: Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs (hunk 5 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs]: Done

**Comment by Smurf-IV** [comment]
@Wagnerp 
Good to go.. Sort conflicts

**Comment by Wagnerp** [comment]
> @Wagnerp
> Good to go.. Sort conflicts

@Smurf-IV That should be good to go

**Pull request merged**